stats/stats/src/update/new_blocks.rs

```diff
@@ -249,7 +252,9 @@ mod tests {
         mock_blockscout(&blockscout).await;
 
         Updater::default().update(&db, &blockscout).await.unwrap();
-        let data = get_chart_int(&db, "newBlocksPerDay").await.unwrap();
+        let data = get_chart_int(&db, "newBlocksPerDay", None, None)
```

Comment:
Nit: I suggest to assign `"newBlocksPerDay"` value to a const variable and just reuse it for all tests. So, if some renaming would be required, we can change only one variable value

Reply:
Agree

Reply:
I made "name" function for trait for that reason, but I'd like to replace it in later PR